std/src/main/scala/cats/std/option.scala

```diff
@@ -58,4 +58,17 @@ trait OptionInstances {
       def eqv(x: Option[A], y: Option[A]): Boolean =
         x.fold(y == None)(a => y.fold(false)(ev.eqv(_, a)))
     }
+
+  implicit def optionMonoid[A](implicit ev: Monoid[A]): Monoid[Option[A]] =
```

Comment:
I'm pretty sure you just want to require the `Semigroup[A]` since you don't need to use its `empty` method.